### ISSUE TITLE
Show number of files, selected files in Diff tab

### ISSUE THREAD

**Issue by webmaster33** [gitextensions/gitextensions#3664]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
Diff tab doesn't show any information about number of files listed in Diff tab.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
SUGGESTION:
Diff tab should show 3 number about listed files:
number of Selected files / number of Filtered files / number of All files in diff

Maybe could be displayed in short format like: S:5 F:100 A:900

The info could be possible to display in a new status bar.
Or if we don't want to lose GUI space, then it can be displayed through tooltip, after the filename 
(Example: admin/index.php  - S:5 F:100 A:900). In this case tooltip should be always shown on mouseover over any filename (not just when a filename is not fully shown).

**Depending on issue:**
This feature may help using following features:
- Sorting files in Diff tab #3662
- Filtering by status in Diff tab #3663
- Cherry picking multiple file changes in Diff tab #3665

**Which versions of GIT and which versions of Windows are affected by this issue? Did this work in previous versions of our tool?**
<2.49.03

**Comment by gregsohl** [comment]
+1 - I needed this feature just yesterday.

**Comment by rmsmani** [comment]
HI @gregsohl,

I hv made changes to the https://github.com/gitextensions/gitextensions/blob/master/GitUI/UserControls/FileStatusList.cs
and the snapshot of the changes are given below.

![image](https://user-images.githubusercontent.com/28680638/51108543-abc36f00-1846-11e9-9319-53897cac07ad.png)


Currently I had included 3 labels in the StatusStrip which is added at the bottom to show
1. Total files in the list
2. Selected files count
3. Filtered files count

As of now I made the design changes.  Check and let'me know to proceed with the coding for setting up the values to these fields.

**Comment by RussKie** [comment]
@rmsmani please raise a PR so those interested can pull it down and try.

I would remove "Files" and leave just "Total: {x} / Selected: {x} / Filtered: {x}".
I would also show "Selected" and "Filtered" only if there are selected or filtered files.

**Comment by gerhardol** [comment]
I would rather have a tooltip similar to the artificial commit when hovering "Diff with"
Having it in a statusbar is unclear for merge commits/multiple selections.

**Comment by GintasS** [comment]
Abandoning this because of age and lack of discussion. Some sketches how the window should look have been provided, but no one seems to work on this.

**Comment by klorinczi** [comment]
Why abandoning features, which are useful?

**Comment by gerhardol** [comment]
> Why abandoning features, which are useful?

Because there were almost 1000 issues opened, it was impossible to find relevant issues and features.

If someone want to implement something like this, make a proposal in this or a new issue and submit a PR.
Personally, I do not believe this is adding more info than it hides for me. (tooltip is fine)
